app/models/responsible_body.rb

```diff
@@ -232,11 +231,11 @@ def next_school_sorted_ascending_by_name(school)
   end
 
   def orders_managed_centrally?
-    who_will_order_devices == 'responsible_body'
+    default_who_will_order_devices_for_schools == 'responsible_body'
   end
 
   def orders_managed_by_schools?
-    %w[school schools].include?(who_will_order_devices)
+    %w[school schools].include?(default_who_will_order_devices_for_schools)
   end
```

Comment:
Both these methods could return `true`. Are we sure that they are being used correctly?

Reply:
I have refactored/renamed these 2 like this:
```
  def responsible_body_will_order_devices_for_schools_by_default?
    default_who_will_order_devices_for_schools == 'responsible_body'
  end

  def schools_will_order_devices_by_default?
    default_who_will_order_devices_for_schools == 'school'
  end

```